ocw_import/api.py

```diff
@@ -268,6 +289,9 @@ def import_ocw2hugo_sitemetadata(
         "content": [content.text_id for content in instructor_contents],
     }
 
+    metadata["term"] = course_data["term"]
+    metadata["year"] = course_data.get("year")
```

Comment:
Minor thing, wondering if we should use `course_data.get()` consistently in this function, or is `course_data[]` used when it is guaranteed to be present?

Reply:
Right, `term` exists in both the legacy and updated format so I used `[]` for that